newIDE/electron-app/app/PreviewWindow.js

```diff
@@ -71,10 +73,11 @@ const openPreviewWindow = ({
 
     previewWindows.push(previewWindow);
 
-    previewWindow.on('closed', event => {
+    previewWindow.on('closed', closeEvent => {
       previewWindows = previewWindows.filter(
         otherPreviewBrowserWindow => otherPreviewBrowserWindow !== previewWindow
       );
+      openEvent.sender.send('preview-window-closed');
```

Comment:
locally, I use the event that opened the preview in order to send back a message to the sender. Seems to work well